tile-generator

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+const tileGenerator = require('./');
+const assert = require('assert');
+const argv = require('yargs')
+    .usage("$0 --minzoom 0 --maxzoom 10 --bbox east,south,west,north")
```

Comment:
The ordering here is incorrect. Bounding boxes are specified in W, S, E, N order

Reply:
I can't believe I stuffed that up, I mixed up my east and west!